packages/experimental/Badge/src/Badge.types.ts

```diff
@@ -132,6 +132,16 @@ export interface BadgeTokens extends BadgeCoreTokens, BadgeConfigurableProps {
    */
   iconWeight?: number;
 
+  /**
+   * When isRTL - applies Badge from the left side
+   */
+  rtl?: BadgeTokens;
```

Comment:
I don't think we need both - we can default to one state and then apply the other when it's true? For example the default is ltr but if rtl is true then we apply what we need for rtl and reassign anything for ltr to be undefined, or something like that.

Reply:
I'll try to do with undefined

Reply:
It worked :)